### PR TITLE
Remove duplicated unnecessary codes

### PR DESCRIPTION
**Pull request by chozzz**
### Background
Long support for maintainability, it also helps for others to read the code.

### Changes
- Remove multiple handler for `args.debug`
- Remove Config being instantiated twice.

### Documentation
Not needed

### Test Plan
N/A

### PR Quality Checklist
- [x] My pull request is atomic and focuses on a single change.
- [ ] I have thouroughly tested my changes with multiple different prompts.
- [x] I have considered potential risks and mitigations for my changes.
- [x] ~I have documented my changes clearly and comprehensively.~
- [x] I have not snuck in any "extra" small tweaks changes

### CODE REVIEW

**Comment by nponeccop** [comment]
@chozzz There are conflicts now.

File: scripts/main.py
Comment (nponeccop) [inline scripts/main.py:318]: Whitespace is fixed in separate whitespace-only PRs, so revert this line

**Review by nponeccop — CHANGES_REQUESTED** [review]

**Comment by chozzz** [comment]
Yeah, its reported here - https://github.com/Torantulino/Auto-GPT/issues/823

File: scripts/main.py
Comment (chozzz) [reply scripts/main.py:318]: Reverted

**Review by jaspythonis — APPROVED** [review]

**Comment by nponeccop** [comment]
The problem is that the new PR is unmergeable. The maintainer only wants focused PRs fixing a single issue.

**Comment by chozzz** [comment]
This is a one-liner refactor that doesn't fix anything, and imo it makes more sense for this change to be reviewed with the issues that `master` is facing since they're related. Hence, I raised another PR #888  to fix the issues in `master`.

I'd rather the maintainer to consider the other PR which can unblock `master` from major issues thats breaking it.

**Comment by nponeccop** [comment]
It does and it has been already reviewed and queued to merge. You broke the pipeline. I suggest reopening.

**Review by nponeccop — DISMISSED** [review]
Thanks for the reopening

**Review by richbeales — APPROVED** [review]

**Comment by nponeccop** [comment]
@chozzz There are conflicts now

**Comment by chozzz** [comment]
@nponeccop Thanks for letting me know - ive fixed the conflict

**Review by nponeccop — APPROVED** [review]

**Review by richbeales — APPROVED** [review]

**Pull request merged**
